doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst

```diff
@@ -106,18 +108,46 @@ If the verification fails, the boot process stops.
 This way, the immutable bootloader can guarantee that the next image in the boot sequence will not start up if it has been tampered with in any way.
 For example, if an attacker attempts to take over the device by altering the firmware, the device will not boot, and thus not run the infected code.
 
-The immutable bootloader is locked to the flash memory and cannot be modified or deleted without erasing the entire device.
-For the |NSIB|, see :ref:`Locking the flash memory <bootloader_rot>` for more information.
+More specifically, the immutable bootloader performs the following steps when it runs:
```

Comment:
NSIB does exactly these steps.
However,  other Immutable bootloaders can do other things as well.
I agree with these are a minimum amount of things an immutable bootloader must do, but maybe phrase this so that it is not "the list is all it does and nothing more", but instead "These are the things it does, but it could also do more depending on bootloader and configuration"